modules/apps/object/object-api/src/main/java/com/liferay/object/field/util/ObjectFieldUtil.java

```diff
@@ -29,14 +29,14 @@ public class ObjectFieldUtil {
 
 	public static ObjectField createObjectField(
 		long listTypeDefinitionId, String businessType, String dbColumnName,
-		String dbType, boolean indexed, boolean indexedAsKeyword,
+		String dbType, String externalReferenceCode, boolean indexedAsKeyword,
 		String indexedLanguageId, String label, String name, boolean required,
-		boolean system) {
+		boolean system, boolean indexed) {
```

Comment:
@seltong please do not change the order of the `indexed` property.. We should follow the same order from the service.xml